docs/conceptual/gpu-arch/mi200-performance-counters.md

```diff
@@ -31,7 +31,7 @@ the names require further evaluation.
 
 ### Command processor
 
-The CPCs are further classified into fetcher (CPF) and compute (CPC).
+The command processors (CPs) are further classified into fetcher (CPF) and compute (CPC).
```

Comment:
@SwRaw FYI

Reply:
We cannot use the CPC abbreviation because it has a different meaning (Command Processor - Compute). The repetitive usage of the command processor was perhaps to explain the further division into command processor - fetcher and command processor- compute. We can use "CP counters" instead.

Reply:
@SwRaw  just to be sure, you're saying we should change "The command processors (CPs)..." to "The command processor (CP) counters..."?

Reply:
@LisaDelaney Currently you have:

Command processor
The command processors (CPs) are further classified into fetcher (CPF) and compute (CPC).

It should be:

Command processor (CP)
The CP counters are further classified into fetcher and compute.
--------------------------------------------

Note that I am not using CPF and CPC in the description because I want it to be in the sub-heading instead to maintain a uniform format like other headings and also to make it easier for the user to navigate to the correct table based on the heading.